app/components/Editor.js

```diff
@@ -70,8 +75,18 @@ export default class Editor extends React.Component {
         return this.redactor.editor.getElement().find(selector).nodes;
       },
       getCurrentLineText: () => {
-        // Returns the node where the cursor currently is. Typically a paragraph if no formatter, otherwise the closest formatted element
+        // Returns the text content of the node where the cursor currently is. Typically a paragraph if no formatter, otherwise the closest formatted element
         let node = this.redactor.selection.getCurrent();
+
+        // If the node is a figure, remove the <figure> tag and return the child nodes
+        // This is to allow for saving of images, videos, audio when loading Filesafe elements
+        if (node.nodeName === "FIGURE") {
+          let inserted = node.innerHTML;
```

Comment:
Not a blocker for this release, but as a general note, always use `const` instead of `let` if you're not reassigning the variable.